spec/shared/default_rspec_language_config_context.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+RSpec.shared_context 'with default AllCops.RSpec.Language config', :config do
```

Comment:
This will be needed by custom cops if they have specs for DSL customization.
Can't think why this would be necessary for release 2.0 though.
We don't enforce using alias configuration, and custom cops can continue to use their own configuration until someone decides to follow the mainstream, and would like to contribute by extracting this somewhere to `lib/`.